adapter/internal/discovery/xds/server.go

```diff
@@ -76,6 +76,7 @@ var (
 	apiToVhostsMap           map[string]map[string]struct{} // API_UUID -> VHosts set (for un-deploying APIs from API-CTL)
```

Comment:
can you clean up apiUUIDToGatewayToVhosts and apiToVhostsMap too with this PR? if they are not used for APK